cmd/operator-sdk/bundle/cmd.go

```diff
@@ -44,10 +44,22 @@ Operator Lifecycle Manager:
 https://github.com/operator-framework/operator-lifecycle-manager
 `,
 	}
+	return cmd
+}
 
+func NewCmdLegacy() *cobra.Command {
```

Comment:
Could we add a deprecated note/comment?

Reply:
This isn't deprecated yet, and the linter will complain if an exported function is used while deprecated. Ditto below.

Reply:
Regards the lint we can use nolint in this scenario, however, the exported one I agree that we should not add the note to not be visible in the godocs. 

However, I understand that we also agreed in the -2 E.g meeting ago use the Go deprecation annotation to make simple and very visible since it shows with strikethrough on the func and let us know very easy what should or not receiving a new logic and is used injust for the legacy and etc in order to eep the code maintained and cleanup all when the legacy one is no longer be supported.  

IMO the deprecated comment would be more important to be used in the funcs which are just used by `NewCmdLegacy()` and has no utility for the new layout at all.

Reply:
Regards the lint we can use nolint in this scenario, however,  I agree that we should not add the note to not be visible in the godocs in the exported ones. 👍